ProceduralForest/Structures/filesystem.h

```diff
@@ -0,0 +1,52 @@
+#ifndef FILESYSTEM_H
```

Comment:
why does this exist? the only place it is used leads to a files not found

Reply:
it was taken from learnopengl